libbeat/scripts/Makefile

```diff
@@ -40,6 +40,7 @@ SYSTEM_TESTS?=false
 GOX_OS?=linux darwin windows solaris freebsd netbsd openbsd
 DOCKER_COMPOSE?=docker-compose -f docker-compose.yml
 GOPACKAGES_COMMA_SEP=$(subst $(space),$(comma),$(strip ${GOPACKAGES}))
+PYTHON_ENV=${BUILD_DIR}/python-env/
```

Comment:
Nice! Would it be OK if I change this to `PYTHON_ENV?=${BUILD_DIR}/python-env/`? I would like to be able to set this variable in my environment to move the env directory off of a shared drive because virtualenv has issues when it cannot create symlinks on a CIFS mount.